README.md

```diff
@@ -377,10 +377,10 @@ Other Style Guides
     ```javascript
     const foo = document.querySelectorAll('.foo');
 
-    // good
+    // bad
     const nodes = Array.from(foo);
```

Comment:
this is a good example - use of `Array.from` is very good, because of its mapper argument.